test-e2e/tests/account_test.ts

```diff
@@ -98,13 +102,21 @@ Scenario('I get a wrong password warning', async ({ I }) => {
   ]);
 });
 
-Scenario('I can toggle to view/hide my password', async ({ I }) => {
+Data(configs).Scenario('I can toggle to view/hide my password', async ({ I, current }) => {
+  if (!current.canEditEmail) {
+    return;
+  }
+  loginContexts[current.config.label] = await I.beforeAccount(current.config, loginContexts[current.config.label], firstName, lastName);
   I.amOnPage(constants.accountsUrl);
   I.click(editAccount);
   await passwordUtils.testPasswordToggling(I, 'confirmationPassword');
 });
 
-Scenario('I can reset my password', async ({ I }) => {
+Data(configs).Scenario('I can reset my password (ResetLink)', async ({ I, current }) => {
+  if (current.resetPasswordType !== 'resetlink') {
```

Comment:
Is there a test for reset password 'direct'?

Reply:
there is no way to write a test for this since the sandbox mails are received on mailhog.